src/main/java/com/microfocus/lrc/jenkins/TestRunBuilder.java

```diff
@@ -546,73 +513,45 @@ public void perform(
         }
 
         ServerConfiguration serverConfiguration = createServerConfiguration(descriptor, run, launcher);
-        ProxyConfiguration proxyConfiguration = ProxyConfigurationFactory.createProxyConfiguration(
+        ProxyConfiguration proxyConfiguration = ConfigurationFactory.createProxyConfiguration(
                 serverConfiguration.getUrl(),
                 descriptor.useProxy,
                 descriptor.proxyHost,
                 descriptor.proxyPort,
                 descriptor.proxyUsername,
                 Secret.fromString(descriptor.proxyPassword).getPlainText(),
-                logger
+                this.loggerProxy
         );
         serverConfiguration.setProxyConfiguration(proxyConfiguration);
 
-        JsonObject buildResult = new JsonObject();
-        buildResult.addProperty(Constants.PROJECTID, this.getProjectIdAtRunTime(run, launcher));
-        buildResult.addProperty(Constants.SENDEMAIL, this.isSendEmail());
-        String resultStr = buildResult.toString();
-        FilePath resultFile = workspace.child("build_result_" + run.getId());
         int testIdVal = Integer.parseInt(this.getTestIdAtRunTime(run, launcher));
-        TestRunOptions opt = new TestRunOptions(testIdVal, this.sendEmail);
         Map<String, String> envVarsObj = this.readConfigFromEnvVars(run, launcher);
+        TestRunOptions opt = new TestRunOptions(
+                testIdVal,
+                this.sendEmail,
+                Boolean.parseBoolean(envVarsObj.get(OptionInEnvVars.SRL_CLI_SKIP_LOGIN.name())),
+                Boolean.parseBoolean(envVarsObj.get(OptionInEnvVars.LRC_SKIP_PDF_REPORT.name())),
+                Boolean.parseBoolean(envVarsObj.get(OptionInEnvVars.LRC_DEBUG_LOG.name()))
+        );
 
         RunTestCallable callable = new RunTestCallable(
+                logger,
                 serverConfiguration,
-                listener,
```

Comment:
`resultStr` and `resultFile` are not needed, since we do not need handle interrupted run results